meta-resin-common/conf/distro/include/resin-os.inc

```diff
@@ -74,3 +74,4 @@ RESIN_FLASHER_FLAG_FILE = "resin-image-flasher"
 INITRAMFS_IMAGE = "resin-image-initramfs"
 INITRAMFS_IMAGE_BUNDLE = "1"
 KERNEL_INITRAMFS = "-initramfs"
+INITRAMFS_TASK = ""
```

Comment:
I don't understand how this was introduced. Is it a newer poky morty change that we do not have in any of our resin-<board> yet?